kb_text_shape.h

```diff
@@ -19476,7 +19472,7 @@ static kbts_begin_cluster_result kbts_BeginCluster(kbts_shape_state *ShapeState,
         kbts_feature *Vatu = Config->Vatu;
 
         // pstf forms come last, then blwf.
-        kbts_feature *SectionFeature = Config->Pstf;
+        kbts_feature *SectionFeature = Pstf;
```

Comment:
Following the pattern os the `kbts_feature *Locl = Config->Locl;` where `Locl` is used.